client/app/directives/right-click.directive.js

```diff
@@ -14,14 +14,10 @@
 
         function link(scope, element, attrs) {
```

Comment:
@maxfadeev 
Is this directive still required?
I see that it's no longer being used in _client/app/components/files/files.html_